argus.yaml

```diff
@@ -129,43 +125,53 @@ request:
     # maxTTL specifies the cap for the TTL of items when values are specified.
     maxTTL: "24h"
 
-# jwtValidator provides Bearer auth configuration
-jwtValidator:
-  keys:
-    factory:
-      uri: "http://sample-jwt-validator-uri/{keyId}"
-    purpose: 0
-    updateInterval: 604800000000000
-
-# capabilityCheck provides the details needed for checking an incoming JWT's
-# capabilities.  If the type of check isn't provided, no checking is done.  The
-# type can be "monitor" or "enforce".  If it is empty or a different value, no
-# checking is done.  If "monitor" is provided, the capabilities are checked but
-# the request isn't rejected when there isn't a valid capability for the
-# request. Instead, a message is logged.  When "enforce" is provided, a request
-# that doesn't have the needed capability is rejected.
-#
-# The capability is expected to have the format:
-#
-# {prefix}{endpoint}:{method}
-#
-# The prefix can be a regular expression.  If it's empty, no capability check
-# is done.  The endpoint is a regular expression that should match the endpoint
-# the request was sent to. The method is usually the method of the request, such as
-# GET.  The accept all method is a catchall string that indicates the capability
-# is approved for all methods.
-# (Optional)
-#capabilityCheck:
-# # type provides the mode for capability checking.
-# type: "monitor"
-# # prefix provides the regex to match the capability before the endpoint.
-# prefix: "xmidt"
-# # acceptAllMethod provides a way to have a capability that allows all
-# # methods for a specific endpoint.
-# acceptAllMethod: "all"
-# # endpointBuckets provides regular expressions to use against the request
-# # endpoint in order to group requests for a metric label.
-# endpointBuckets:
-#   - "store\\b"
-#   - "store/.*\\b"
-
+bascule:
```

Comment:
I changed it to bascule as most of the components of each profile are driven by bascule and as during our last talk John mentioned the idea of bascule profiles. I agree with you that folks wouldn't really know about it... Since capabilityChecks and the future accessLevel are authorizing instead of simply authenticating, could we do a more umbrella term like "security" or something else?

Reply:
I'm good with security - might be good to get buy in from the team too. 👍